src/client/store/theme/types.ts

```diff
@@ -21,4 +22,9 @@ export interface SetCurrentThemeAction extends Action {
   payload: UserTheme;
 }
 
-export type ThemeAction = SetThemesAction | SetCurrentThemeAction;
+export interface SetRemoveThemeAction extends Action {
+  type: 'SET_REMOVE_THEME';
```

Comment:
Минорная вкусновщина, но может 
```suggestion
  type: 'REMOVE_THEME';
```